src/aead/quic.rs

```diff
@@ -82,7 +81,7 @@ const SAMPLE_LEN: usize = super::TAG_LEN;
 pub struct Algorithm {
     init: fn(key: &[u8], cpu_features: cpu::Features) -> Result<KeyInner, error::Unspecified>,
 
-    new_mask: fn(key: &KeyInner, sample: Block) -> [u8; 5],
+    new_mask: fn(key: &KeyInner, sample: [u8; BLOCK_LEN]) -> [u8; 5],
```

Comment:
We have a constant `SAMPLE_LEN` that we should use instead. Ditto for the other `sample:` stuff.

Reply:
Alright, was following patterns here. Will use `SAMPLE_LEN`.